packages/react-styled-ui/src/Text/index.js

```diff
@@ -1,23 +1,32 @@
 import React, { forwardRef } from 'react';
-import Box from '../Box';
+import PseudoBox from '../PseudoBox';
+import useColorMode from '../useColorMode';
 import useTheme from '../useTheme';
+import getColorPalette from '@trendmicro/styled-ui-theme/build/color-palette';
 
 const Text = forwardRef((
   {
+    as,
     size,
     ...rest
   },
   ref,
 ) => {
   const { fontSizes } = useTheme();
+  const { colorMode } = useColorMode();
+  const markedBgColor = getColorPalette(colorMode).get('background.marked');
   const sizeProps = {};
   if (size !== undefined && Object.prototype.hasOwnProperty.call(fontSizes, size)) {
     sizeProps.fontSize = size;
     sizeProps.lineHeight = size;
   }
+  if (as === 'mark') {
+    sizeProps.backgroundColor = markedBgColor;
```

Comment:
The background color is nothing to do with the size. How about adding a new `colorProps` rather than using the `sizeProps`?